### PR TITLE
Create the jupyter notebook for the examples

### LINKED ISSUE #47: JOSS REVIEW: Examples
- [ ] Please consider merging README markdown files and PYTHON scripts into a Jupyter notebook.
They are easier to read, especially on GitHub. You can also take the user through the step of installing the required Python dependencies...

- [ ] Please don't expect user to have PETSc installed in their HOME directory as default...

- [ ] Jupyter notebooks would also help to visualize the outcome of the models / Benchmarks. As it is now, users must run the model themselves to see the result and they don't have a mean of comparison. That is also where you should compare with other codes (I2VES, UNDERWORLD, STAGYY)

### PR DESCRIPTION
**Pull request by aguspesce**
I'm making a Jupyter notebook for each of the examples.
Each notebook generates the input files, gives instructions to run the model using Mandyoc, and plots the results.

I am taking into account the comments of the reviewer:

- [x] Please consider merging README markdown files and PYTHON scripts into a Jupyter notebook. They are easier to read, especially on GitHub. You can also take the user through the step of installing the required Python dependencies...
- [x] - Please don't expect user to have PETSc installed in their HOME directory as default...
- [x] - Jupyter notebooks would also help to visualize the outcome of the models / Benchmarks. As it is now, users must run the model themselves to see the result and they don't have a mean of comparison. That is also where you should compare with other codes (I2VES, UNDERWORLD, STAGYY)

addresses #47

### CODE REVIEW

**Comment by aguspesce** [comment]
@jamisonassuncao, @victorsacek, @rafaelmds,
I've finished passing all the script in a Jupyter notebook.  Can you give them a reviewer?

**Comment by rafaelmds** [comment]
@aguspesce We will close JOSS issues manually. Could you change "fix" to something like "addresses".

**Comment by rafaelmds** [comment]
- remove white space in file name [comparing plot_max_surface.ipynb](https://github.com/ggciag/mandyoc/pull/58/files#diff-0252c355e0522d172d413fd8dc6989dc875f91608a45c67006ae1aed6aef3c75)

File: examples/Crameri2012_case2/comparing plot_max_surface.ipynb [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    "\n",
+    "    if step == 0 and int(surface_topo_mandyoc) == 0:\n",
+    "        surface_topo_mandyoc = 1.5e5\n",
+    "\n",
+    "    # read time\n",
+    "    with open(f\"time_{step}.txt\") as time_file:\n",
+    "        time_info = time_file.readline()\n",
+    "        _t = float(time_info.split(\":\")[-1])\n",
+    "        times = np.append(times, _t)\n",
+    "        \n",
```
Comment (rafaelmds) [inline examples/Crameri2012_case2/comparing plot_max_surface.ipynb:105]:
```suggestion
    "\n",
    "arg_sorted = np.argsort(times)\n",
    "times = times[arg_sorted]\n",
    "surface_topo_mandyoc = surface_topo_mandyoc[arg_sorted]\n",
    "\n",
```

surface_filenames are not being sorted correctly. This fix ordering to plot points in the correct order (time ordered).

**Review by rafaelmds — CHANGES_REQUESTED** [review]

**Review by rafaelmds — APPROVED** [review]

**Comment by aguspesce** [comment]
@victorsacek: I fixed the suggestion you sent me by mail

**Pull request merged**